src/snakemake_interface_logger_plugins/settings.py

```diff
@@ -8,6 +8,21 @@
 import snakemake_interface_common.plugin_registry.plugin
 
 
+from abc import ABC
+from typing import Optional, Sequence
+
+
+class LoggerSettingsInterface(ABC):
```

Comment:
The convention in the other plugin interfaces is to name these by the originating settings class, i.e. `OutputSettingsLoggerInterface`